addons/mail/static/tests/qunit_suite_tests/components/message_in_reply_to_tests.js

```diff
@@ -0,0 +1,113 @@
+/** @odoo-module **/
+
+import { start, startServer } from "@mail/../tests/helpers/test_utils";
+
+import { nextTick, patchWithCleanup } from "@web/../tests/helpers/utils";
+
+QUnit.module("mail", {}, function () {
+    QUnit.module("components", {}, function () {
+        QUnit.module("message_in_reply_to_tests");
+
+        QUnit.test(
+            "click on message in reply to highlight the parent message",
+            async function (assert) {
+                assert.expect(1);
+
+                const pyEnv = await startServer();
+                const mailChannelId1 = pyEnv["mail.channel"].create({ name: "general" });
+                const mailMessageId1 = pyEnv["mail.message"].create({
+                    body: "Hey lol",
+                    message_type: "comment",
+                    model: "mail.channel",
+                    res_id: mailChannelId1,
+                });
+                const mailMessageId2 = pyEnv["mail.message"].create({
+                    body: "Response to Hey lol",
+                    message_type: "comment",
+                    model: "mail.channel",
+                    parent_id: mailMessageId1,
+                    res_id: mailChannelId1,
+                });
+                const { click, env, openDiscuss } = await start({
+                    discuss: {
+                        params: {
+                            default_active_id: `mail.channel_${mailChannelId1}`,
+                        },
+                    },
+                });
+                // TODO-DISCUSS-REFACTORING: remove when active id will be handle.
+                env.services["mail.messaging"].setDiscussThread(mailChannelId1);
+                await openDiscuss();
```

Comment:
Pushed a 3rd commit to fix the tests:
- `setDiscussThread()` before `openDiscuss()` (instead of `click()` on sidebar channel item)
- set a name of channel to make sure `AutogrowInput` props validation is happy (`value` and `title` were not String but `false`)